code/src/components/AddTask.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import styled from "styled-components";
+
+import tasks from "../reducers/tasks";
+
+const AddTask = () => {
+  const [input, setInput] = useState("");
+
+  const dispatch = useDispatch();
+
+  const onAddTask = () => {
+    dispatch(tasks.actions.addTask(input));
+    setInput(""); // -- clears the input field when submitted
+  };
+
+  const handleKeyPress = (event) => {
```

Comment:
I like this solution, we will have to use it on our current project since we are not doing a form! =D If you do a form with a submit button you dont need this! =)